### PR TITLE
Add travel funtion

### PR DESCRIPTION
**Pull request by jiratQ**
This function can't use with bot (sometimes can sometime can't don't know why pls review)
must use (with user that connect in voice channel and target in voice channel too) if not will occur error 
P.S. I think we can't fix both of 2 problem. I just want you to reviews.

### CODE REVIEW

**Review by namirinz — DISMISSED** [review]
Well done!

File: src/utils/utils.py [hunk 1]
```diff
@@ -27,3 +27,5 @@ async def guess_command(client, message, commands):
         similar_commands = sorted(
             similar_commands, key=lambda l: l[1], reverse=True)
         await message.channel.send(f'คุณกำลังจะพิมพ์ {prefix}{similar_commands[0][0]} หรือเปล่า')
+
+
```
Comment (namirinz) [inline src/utils/utils.py:31]: Nice

File: src/audio/audio.py [hunk 2]
```diff
@@ -2,21 +2,14 @@
...
 from asyncio import TimeoutError
 import os, sys
-
+from src.utils.vc import get_PATH_ffmpeg
 from src.utils.utils import config
 from src.audio.tts import repeat
 
 PATH_mp3 = config.audio.PATH_mp3
-
+PATH_ffmpeg = get_PATH_ffmpeg()
```
Comment (namirinz) [inline src/audio/audio.py:10]: Well Done!

File: src/utils/utils.py (hunk 1 above)
Comment (phusitsom) [reply src/utils/utils.py:31]: Thanks :heart:

**Review by phusitsom — APPROVED** [review]

**Pull request merged**